EIPS/eip-3529.md

```diff
@@ -0,0 +1,151 @@
+| -- | -- | -- | -- | -- | -- | -- | -- | 
+| `0x60006000556000600055` | 212 | 0| 0 | 0 |  0 |  |  212 |
+| `0x60006000556001600055` | 20112 | 0| 0 | 0 |  1 |  |  20112 |
+| `0x60016000556000600055` | 20112 | 19900| 0 | 1 |  0 |  |  212 |
+| `0x60016000556002600055` | 20112 | 0| 0 | 1 |  2 |  |  20112 |
+| `0x60016000556001600055` | 20112 | 0| 0 | 1 |  1 |  |  20112 |
+| `0x60006000556000600055` | 3012 | 15000| 1 | 0 |  0 |  |  -11988 |
+| `0x60006000556001600055` | 3012 | 2800| 1 | 0 |  1 |  |  212 |
+| `0x60006000556002600055` | 3012 | 0| 1 | 0 |  2 |  |  3012 |
+| `0x60026000556000600055` | 3012 | 15000| 1 | 2 |  0 |  |  -11988 |
+| `0x60026000556003600055` | 3012 | 0| 1 | 2 |  3 |  |  3012 |
+| `0x60026000556001600055` | 3012 | 2800| 1 | 2 |  1 |  |  212 |
+| `0x60026000556002600055` | 3012 | 0| 1 | 2 |  2 |  |  3012 |
+| `0x60016000556000600055` | 3012 | 15000| 1 | 1 |  0 |  |  -11988 |
+| `0x60016000556002600055` | 3012 | 0| 1 | 1 |  2 |  |  3012 |
+| `0x60016000556001600055` | 212 | 0| 1 | 1 |  1 |  |  212 |
+| `0x600160005560006000556001600055` | 40118 | 19900| 0 | 1 |  0 |  1 |  20218 |
+| `0x600060005560016000556000600055` | 5918 | 17800| 1 | 0 |  1 |  0 |  -11882 |
+
+### With reduced refunds
+
+If refunds were to be partially removed, by changing SSTORE_CLEARS_SCHEDULE from 15000 to 4800 (and removing selfdestruct refund) this would be the comparative table.
+
+| Code | Used Gas | Refund | Original | 1st | 2nd | 3rd | Effective gas (after refund)
+| -- | -- | -- | -- | -- | -- | -- | -- | 
+| `0x60006000556000600055` | 212 | 0| 0 | 0 |  0 |  |  212 |
+| `0x60006000556001600055` | 20112 | 0| 0 | 0 |  1 |  |  20112 |
+| `0x60016000556000600055` | 20112 | 19900| 0 | 1 |  0 |  |  212 |
+| `0x60016000556002600055` | 20112 | 0| 0 | 1 |  2 |  |  20112 |
+| `0x60016000556001600055` | 20112 | 0| 0 | 1 |  1 |  |  20112 |
+| `0x60006000556000600055` | 3012 | 4800| 1 | 0 |  0 |  |  -1788 |
+| `0x60006000556001600055` | 3012 | 2800| 1 | 0 |  1 |  |  212 |
+| `0x60006000556002600055` | 3012 | 0| 1 | 0 |  2 |  |  3012 |
+| `0x60026000556000600055` | 3012 | 4800| 1 | 2 |  0 |  |  -1788 |
+| `0x60026000556003600055` | 3012 | 0| 1 | 2 |  3 |  |  3012 |
+| `0x60026000556001600055` | 3012 | 2800| 1 | 2 |  1 |  |  212 |
+| `0x60026000556002600055` | 3012 | 0| 1 | 2 |  2 |  |  3012 |
+| `0x60016000556000600055` | 3012 | 4800| 1 | 1 |  0 |  |  -1788 |
+| `0x60016000556002600055` | 3012 | 0| 1 | 1 |  2 |  |  3012 |
+| `0x60016000556001600055` | 212 | 0| 1 | 1 |  1 |  |  212 |
+| `0x600160005560006000556001600055` | 40118 | 19900| 0 | 1 |  0 |  1 |  20218 |
+| `0x600060005560016000556000600055` | 5918 | 7600| 1 | 0 |  1 |  0 |  -1682 |
+
+## The mutex usecase
+
+There are two typical ways to implement mutexes: '0-1-0' and '1-2-1. Let's see how they differ
+
+- '0-1-0':
+  - Istanbul: 1612
+  - Berlin: 212
+  - EIP-3403: 1112
+  - This EIP: 212
+- '1-2-1':
+  - Istanbul: 1612
+  - Berlin: 212
+  - EIP-3403: 3012
+  - This EIP: 212
+
```

Comment:
This isn't a valid EIP section.  Consider converting this into a test case, or putting it as a subheading in the Rationale section.
```suggestion
```